projects/plugins/super-cache/tests/e2e/specs/activation.test.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, beforeAll, expect, test } from '@jest/globals';
+import { readDockerFile, dockerExec } from '../lib/docker-tools';
+import { readPluginFile } from '../lib/plugin-tools';
+import { resetEnvironmnt, wpcli } from '../lib/wordpress-tools';
+
+describe( 'Plugin Activation', () => {
+	beforeAll( async () => {
+		await resetEnvironmnt();
+		await wpcli( 'plugin', 'activate', 'wp-super-cache' );
+	} );
+
+	test( 'Ensure wp-config.php is updated when activated', async () => {
+		const config = await readDockerFile( '/var/www/html/wp-config.php' );
+
+		expect( /define\(\s*'WP_CACHE'/.test( config ) ).toBe( true );
+		expect( /define\(\s*'WPCACHEHOME'/.test( config ) ).toBe( true );
+	} );
+
+	test( 'Ensure advanced-cache is populated correctly.', async () => {
+		const advancedCache = await readDockerFile( '/var/www/html/wp-content/advanced-cache.php' );
+		const expectedContents = await readPluginFile( 'advanced-cache.php' );
+
+		expect( advancedCache ).toBe( expectedContents );
```

Comment:
Should this also have an empty check to ensure the original file is not moved/deleted?
```suggestion
		expect( advancedCache ).toBe( expectedContents );
		expect( advancedCache ).not.toBe( '' );
```

Reply:
I'm not sure I follow. Do you mean making sure that it didn't delete [the advanced-cache file that ships with the plugin](https://github.com/Automattic/wp-super-cache/blob/trunk/advanced-cache.php)?

Reply:
I guess I am aiming at too specific of a case. 

Consider the scenario that someone has moved `./advanced-cache.php` to `./stab/advanced-cache.php` during a refactor. They didn't correctly adjust the code responsible for copying the file. Now, both of the read calls would return an empty string and the test would pass.

Reply:
Oh, I see. That makes sense. I'll double-check but I believe it'll scream if the file isn't there - but checking for empty content doesn't hurt.

Reply:
Done in d56474dfe7.